JavaScript/Topics/5.Syntax/6.Functions/Video.md

```diff
@@ -0,0 +1,136 @@
+
+
+## Javascript Functions
+
+<iframe  width="560"  height="315"  src="https://www.youtube.com/embed/LAl0vdiPVZQ"  title="YouTube video player"  frameborder="0"  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen></iframe>
+
+**Created by :** [Sarfraz Alam](https://github.com/Sarfraz-droid)
+
+## Transcript
+
+A **JavaScript function** is a block of code designed to perform a particular task. Its basically dividing you program into different parts.
+
+Now a function looks like this..  
+Wait.. lets break it into different parts first
+
+```javascript
+function myFunction(p1, p2) {
+
+return p1 * p2; // The function returns the product of p1 and p2
+
+}
+```
```

Comment:
Please indent and not use not-needed whitespace.
```suggestion
```javascript
function myFunction(p1, p2) {
    return p1 * p2; // The function returns the product of p1 and p2
}
```
```